apps/game_client/lib/game_client_web/live/pages/board/show.ex

```diff
@@ -127,8 +129,8 @@ defmodule GameClientWeb.BoardLive.Show do
       category: entity.category,
       shape: entity.shape,
       name: entity.name,
-      x: entity.position.x / 5 + 1000,
-      y: entity.position.y / 5 + 1000,
+      x: entity.position.x / 5 + 1500,
+      y: entity.position.y / 5 + 1500,
```

Comment:
Why 1500?

Reply:
This 1500 is a scale from the mocked board size, since it went up from 10000 to 15000 we needed to change this too, I'll change this number for a division by 1000 since that's what we're doing basically